README.md

```diff
@@ -97,9 +97,9 @@ Here's what the AOSP's own implementation of ADB does:
 ![cnxn](https://github.com/cstyan/adbDocumentation/raw/master/images/googleCNXN.jpg)  
 Notice that the 8 bytes are the same as the bytes previous to the `host::` bytes 
 in the last hex dump. This is from the `data_length` and `data_crc32` fields 
-being set based on wanting to send `host::` as our data.    
-2. The `host::` system-identity string
-![host](https://github.com/cstyan/adbDocumentation/raw/master/images/googleHost.png)  
+being set based on wanting to send `host::` as our data.
+2. The `host::` system-identity string                       
+   ![host](https://github.com/cstyan/adbDocumentation/raw/master/images/googleHost.png)  
```

Comment:
unless this changes the rendered formatting in a way that you think is useful, lets avoid making unrelated whitespace changes here

Reply:
if the text is viewed in fullscreen the image gets put to the right of the text, altought this was only visible in the editor? I'm sorry i dont use github often so i just thought i did something wrong for it to look like that.
![image](https://github.com/cstyan/adbDocumentation/assets/79465618/69288c41-c99c-4e68-b9d0-c8d5d5933fa1)

Reply:
If we view the rendered form of the markdown as it currently exists in the master branch, the formatting is correct
![2024-03-28-103026_1325x331_scrot](https://github.com/cstyan/adbDocumentation/assets/3246492/a12feebe-9a5e-46c7-b448-c16b0f71303f)


But if we view the rendered markdown for your PR the formatting of the image puts it on the same line as you've shown
![2024-03-28-103049_1523x492_scrot](https://github.com/cstyan/adbDocumentation/assets/3246492/76119a08-7368-4eb3-9327-17512ff0e638)

So we should still be fine just to remove the whitespace changes here.